.gitignore

```diff
@@ -1,6 +1,7 @@
 #  SR User Related   #
 ######################
 cache*/
+data*/
 Logs/
```

Comment:
Got it. Will correct that in a few hrs.

-----Original Message-----
From: "Fernando" <notifications@github.com>
Sent: ‎9/‎26/‎2015 19:58
To: "SiCKRAGETV/SickRage" <SickRage@noreply.github.com>
Cc: "Nils" <nils@familievogels.nl>
Subject: Re: [SickRage] Change default DATA_DIR to PROG_DIR/data (#2561)

In .gitignore:
> @@ -1,6 +1,7 @@
>  #  SR User Related   #
>  ######################
>  cache*/
> +data*/
>  Logs/
need to lowercase here then 
—
Reply to this email directly or view it on GitHub.